### ISSUE TITLE
Feature request: assign QoS policy to openstack_networking_port_v2

### ISSUE THREAD

**Issue by DSpeichert** [terraform-providers/terraform-provider-openstack#246]
### Terraform Version
Terraform v0.11.3

### Affected Resource(s)
Please list the resources as a list, for example:
- openstack_networking_port_v2
- openstack_network_qos_v2 (new if needed?)

### Expected Behavior
It would be great if it was possible to attach pre-existing QoS policies to a port.
Full management of QoS policies does not need to be added at first, those are typically created by admins and shared for use by regular users.

Simply adding `qos_policy_id` on `openstack_networking_port_v2` would be a good first step.
More advanced would be resolving QoS policy ID by its name which requires additional API call and possible implementing a new `openstack_network_qos_v2` resource type.

### Actual Behavior
Feature missing

### Important Factoids
- QoS policies don't seem to be supported in gophercloud :(
- QoS is an extension of Neutron, but available by default
- Only one QoS policy can be applied at a time to a port

### References
A typical request to do that is:
`PUT /v2.0/ports/c7b7c8e3-e32f-4715-96a6-da028abebed6`
```
{"port": {"qos_policy_id": "4c7998b6-f46b-4728-afd2-782618644769"}}
```

**Comment by jtopjian** [comment]
@DSpeichert Thanks for the request. This is possible to do, but support for QoS will need added to the upstream Gophercloud library first.

**Comment by ozerovandrei** [comment]
QoS progress will be tracked in #760

**Comment by ozerovandrei** [comment]
This feature is implemented in #781 and will be available in `1.20.0` version.
